python/ray/data/_internal/planner/plan_read_op.py

```diff
@@ -57,10 +60,15 @@ def get_input_data(target_max_block_size) -> List[RefBundle]:
         assert (
             parallelism is not None
         ), "Read parallelism must be set by the optimizer before execution"
+        start = time.time()
         read_tasks = op._datasource_or_legacy_reader.get_read_tasks(parallelism)
+        end = time.time()
+        logger.info(f"Getting read tasks took: {end-start} seconds")
         _warn_on_high_parallelism(parallelism, len(read_tasks))
 
-        return [
+        # Follow up on the comment below
+        start = time.time()
```

Comment:
nit: do we know if there is any timing util we could use in ray repo?

Reply:
It seems they use `Timer`s occasionally but only really for repeated use in classes rather than for logs